pkg/build/options.go

```diff
@@ -20,13 +20,14 @@ import (
 	"fmt"
 )
 
-type buildStrategy string
+//  ignore package name stutter
+type BuildAndStageStrategy string //nolint:golint
 
 const (
 	// bazelStrategy builds and (optionally) stages using bazel
-	bazelStrategy buildStrategy = "bazel"
-	// makeStrategy builds using make and (optionally) stages using krel
-	makeStrategy buildStrategy = "make"
+	bazelStrategy BuildAndStageStrategy = "bazel"
+	// MakeStrategy builds using make and (optionally) stages using krel
+	MakeStrategy BuildAndStageStrategy = "make"
```

Comment:
nit: did you mean for one to be public and the other not?

Reply:
yes, the other one doesn't need to be exported, it isn't used outside the package